apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -0,0 +1,595 @@
+import {commands as actionCommands} from './commands/actionCommands';
+import {commands as behaviorCommands} from './commands/behaviorCommands';
+import {commands as eventCommands} from './commands/eventCommands';
+import {commands as locationCommands} from './commands/locationCommands';
+import {commands as spriteCommands} from './commands/spriteCommands';
+import {commands as worldCommands} from './commands/worldCommands';
+import {commands as validationCommands} from './commands/validationCommands';
```

Comment:
organizational nit: we should have a `./commands/index.js` file that imports/exports all of these commands files ([example here](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/weblab/dialogs/index.js)). that makes importing these commands clearer, and we can also alias each export (instead of aliasing them here)